store/models.py

```diff
@@ -58,6 +58,7 @@ class Product(DrupalFields, Page):  # type: ignore
     description = RichTextField(blank=True)
     price = models.DecimalField(max_digits=10, decimal_places=2)
     available = models.BooleanField(default=True)
+    is_featured = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
Since this has a default, we don't need null or blank = True
```suggestion
    is_featured = models.BooleanField(default=False)
```